ui/src/playlist/PlaylistShow.js

```diff
@@ -10,7 +10,7 @@ import PlaylistDetails from './PlaylistDetails'
 import PlaylistSongs from './PlaylistSongs'
 import PlaylistActions from './PlaylistActions'
 import { Title, isReadOnly } from '../common'
-
+import { Pagination } from '../common/Pagination'
```

Comment:
I know I told you to use the same Pagination used in other views, but maybe for playlists we should use different `rowsPerPageOptions`, I was thinking maybe `[100, 250, 500]`, with default to 100

Reply:
Yes, `[100,250,500]`  seems a good approach, now all songs only get fetched from API if the playlist contains more than 100 songs